docs/features/analysis.md

```diff
@@ -464,6 +464,38 @@ spec:
               fieldPath: status.canary.weights.canary.podTemplateHash
 ```
 
+!!! important
+    Available since v1.2
+Analysis arguments also support valueFrom for reading any Rollout fields and passing them as arguments to AnalysisTemplate.
+Following example references Rollout status field like podTemplateHash and passing them along to AnalysisTemplate
+
+from the Rollout status
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: guestbook
+  labels:
+    appType: demo-app
+    buildType: nginx-app
+    ...
+    env: dev
+    region: us-west-2
+spec:
+...
+  strategy:
+    canary:
+      analysis:
+        templates:
+        - templateName: args-example
+        args:
+        ...
+        - name: canary-hash
+          valueFrom:
+            fieldRef:
+              fieldPath: status.canary.weights.canary.podTemplateHash
```

Comment:
not sure if `status.canary.weights.canary.podTemplateHash` is a valid path. Could you double check? Thanks.

Reply:
agree..let me update to more useful scenario